frappe/core/doctype/data_import_log/data_import_log.json

```diff
@@ -76,10 +76,22 @@
    "role": "System Manager",
    "share": 1,
    "write": 1
+  },
+  {
+   "create": 1,
+   "delete": 1,
+   "email": 1,
+   "export": 1,
+   "print": 1,
+   "read": 1,
+   "report": 1,
+   "role": "SSport Role",
+   "share": 1,
+   "write": 1
```

Comment:
This came in from some other local customization you have I assume?

Reply:
You are right, i'll remove it!